packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -230,6 +242,25 @@ export class AtomicSearchBox {
       this.activeDescendantElement?.previousElementSibling || this.lastValue
     );
   }
+  private get showSuggestions() {
+    return this.hasSuggestions && this.isExpanded;
+  }
+
+  private get allSuggestionElements() {
```

Comment:
I would probably move this into a private variable instead of a getter to avoid calling `renderItems` too many times